src/manage_sql_nvts.c

```diff
@@ -797,36 +797,6 @@ DEF_ACCESS (nvt_iterator_oid, GET_ITERATOR_COLUMN_COUNT);
  */
 DEF_ACCESS (nvt_iterator_name, GET_ITERATOR_COLUMN_COUNT + 2);
 
-/**
- * @brief Get the cve from an NVT iterator.
- *
- * @param[in]  iterator  Iterator.
- *
- * @return Cve, or NULL if iteration is complete.  Freed by
- *         cleanup_iterator.
- */
-DEF_ACCESS (nvt_iterator_cve, GET_ITERATOR_COLUMN_COUNT + 3);
```

Comment:
I'd really like to see the corresponding SQL removed too, and the offsets of the other
nvt_iterator_* functions adjusted, if that's possible.

Reply:
I plan to change the SQL model in a separate PR. Not only this small one, there is more.

Reply:
OK